frontend/src/tests/import-export.test.js

```diff
@@ -190,4 +185,622 @@ describe("Import/export library functionality", () => {
         const ddahJson = ddahData.map(prepareMinimal.ddah);
         expect(ddahJson).toMatchSnapshot();
     });
+
+    it("Import Instructors from JSON/CSV/XLSX", () => {
+        // import correct instructors from XLSX
+        const normalizedSpreadsheetInstructors = normalizeImport(
+            {
+                fileType: "spreadsheet",
+                data: parseSpreadsheet("instructors_correct.xlsx"),
```

Comment:
Put the fancy `__dirname` logic here, rather than in the `parseSpreadsheet` function. Also, if `parseSpreadsheet` is only used for tests, then it should only exist in the `tests` directory.

Reply:
in this case, should I move the `parseSpreadsheet` to `import-export.test` or create a file which exports it under `tests` directory?

Reply:
There should be a `utils.js` or something like that in the test directory.